packages/edit-site/src/components/sidebar-global-styles-wrapper/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useMemo, useState } from '@wordpress/element';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+import { useViewportMatch } from '@wordpress/compose';
+import {
+	Button,
+	privateApis as componentsPrivateApis,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import GlobalStylesUI from '../global-styles/ui';
+import Page from '../page';
+import { unlock } from '../../lock-unlock';
+import StyleBook from '../style-book';
+import { STYLE_BOOK_COLOR_GROUPS } from '../style-book/constants';
+
+const { useLocation, useHistory } = unlock( routerPrivateApis );
+const { Menu } = unlock( componentsPrivateApis );
+const GLOBAL_STYLES_PATH_PREFIX = '/wp_global_styles';
+
+const GlobalStylesPageActions = ( {
+	isStyleBookOpened,
+	setIsStyleBookOpened,
+} ) => {
+	return (
+		<Menu
+			trigger={
+				<Button __next40pxDefaultSize variant="secondary">
+					{ __( 'Preview' ) }
+				</Button>
+			}
+		>
+			<Menu.RadioItem
+				value
+				checked={ isStyleBookOpened }
+				name="radio-controlled"
```

Comment:
While we're tweaking this dropdown, would it be worth making the radio group name meaningful to this context?